### ISSUE TITLE
Additional Command-Line options

### ISSUE THREAD

**Issue by ethanahlquist** [xournalpp/xournalpp#2070]
Hello, this is my first issue request, so please indicate wrong steps in my actions!

Particularly on Linux, I am interested in additional Command-Line options to use in scripts.

**Describe the solution you'd like**
Here are some of the options I am interested in:

1. Ability to input a background image/pdf into project at launch
2. determine input/output files
3. set auto-save/export on close.  ***(unnecessary)
4. removing save warning  ***(unnecessary)

**Additional context**
These features are intended to provide easier editing and notation of images or screenshots on the fly. 

For example, I might want add notation to an image like this (with options determined quickly through a script): 
![image](https://user-images.githubusercontent.com/43525065/84806941-57806980-afbb-11ea-8e4c-9c54aa5ef622.png)

**Comment by Mazurel** [comment]
Xournalpp contains some of the features that you desire. 

> Ability to input a background image/pdf into project at launch

You can open pdf files with xournalpp and they are ready to edit right away.
If you are using png or jpg files you can use imagemagik's `convert` tool.
Just use `convert file.png file.pdf && xournalpp file.pdf`.

> set output/export locations

There is command line flag `-p {output filename}` or `--create-pdf {output filename}` which exports xopp document as pdf.
There is also `-i {output filename}` and `--create-img {output filename}`  which let you export xopp document as svg or png.
If you are writing a script you can do something like 
`xournalpp $1.xopp && xournalpp -p $1.pdf $1.xopp` (Where $1 is name of file)

Hope I have helped.

**Comment by ethanahlquist** [comment]
Thank you! Sorry for the slow response.

converting with imagemagik does work; however, in the solution,
`convert file.png file.pdf && xournalpp file.pdf` 
There seems to be issues with Xournal++ cutting out portions of the pdf.

This is not an issue with the convert, because when manually resetting 
the background withing Xournal it sizes properly.
&nbsp;
Thanks for the second example, when reading the man pages I was expecting
a GUI to open, rather than just convert .xopp file types.
&nbsp;
Just wondering, if I were to look into addressing some of these features, do
you think Contributors would approve of such changes? 

Much thanks man!

**Comment by fpesari** [comment]
A small but useful command to have would be the ability to convert a .pdf to a .xopp file from the CLI, without having to launch the GUI.

This is useful when you have a bunch of PDFs to annotate, you could just batch convert them and start working on the .xopp files directly.

**Comment by roblem** [comment]
To flesh out https://github.com/xournalpp/xournalpp/issues/2070#issuecomment-657454979 a bit more:

For me, it would be ideal to
1. Add a `--create-xopp` option when opening a `pdf` as in `/usr/bin/xournalpp -i assignment1_student_name.pdf --create-xopp assignment1_student_name.pdf.xopp`
2. Add a `--zoom` option which applies a zoom level when saving the xopp (this way all the batched pdf would open in xournal at the same zoom level) 

So the entire command would be
`/usr/bin/xournalpp -i assignment1_student_name.pdf --create-xopp assignment1_student_name.pdf.xopp --zoom x`
where x is some zoom level xournalpp understands.

**Comment by roblem** [comment]
I agree with https://github.com/xournalpp/xournalpp/issues/2070#issuecomment-657454979: batch conversion to xopp from pdf would be useful.
